doc/api/tls.md

```diff
@@ -995,7 +995,7 @@ argument.
 <!-- YAML
 added: v0.11.13
 changes:
-  - version: REPLACEME
+  - version: v10.15.2
```

Comment:
It seems like this feature has never landed in v10.x (there is a backport for 10.x open. See #24979). I guess we should remove the changes entry until it actually lands?

Reply:
Ah good catch, i'll back that one out